src/modules/fancyzones/editor/FancyZonesEditor/Properties/Resources.resx

```diff
@@ -253,4 +253,7 @@
   <data name="Error_Monitor_Match_Not_Found" xml:space="preserve">
     <value>Match not found ({0})</value>
   </data>
+  <data name="Reset_Layout" xml:space="preserve">
+    <value>Reset</value>

```

Comment:
Let's add a comment to help the localization team.
```suggestion
    <value>Reset</value>
    <comment>as in Reset to a blank value</comment.
```

Reply:
just `>` at the end :)